grammars/grammar-tact.json

```diff
@@ -175,7 +175,7 @@
       ]
     },
     "operator-logic": {
-      "match": "(==|<(?!<)|<=|>(?!>)|>=|\\&\\&|\\|\\||\\:(?!=)|\\?)",
+      "match": "(==|<(?!<)|<=|>(?!>)|>=|\\&\\&|\\|\\||\\:(?!=)|\\?|^\\!(=|!)?)$",
```

Comment:
I think I'd prefer to have explicitly listed operators here

Reply:
yeah, me too, and I found out in here that `&` and `|` are also matched by `operator-binary` rules, so if we write `&&&:{tact}` they will all be colored 😅, so I'm wondering whether we need some more restrict rules to match or keep use loose match but be more careful when we do review, and loose match about `!` gonna be `![!|=]?`

current one can't color operator-logic in codes tho, if keep be stric match, need consider match white space as well